### PR TITLE
Edit user

### PR DESCRIPTION
**Pull request by chris-diesner**

### CODE REVIEW

File: backend/src/test/java/de/neuefische/capstone/backend/controller/UserControllerTest.java [hunk 1]
```diff
@@ -1,56 +1,63 @@
...
 
-@SpringBootTest
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @AutoConfigureMockMvc
 class UserControllerTest {
 
     @Autowired
     private MockMvc mockMvc;
 
+    @MockBean
```
Comment (MJaschewski) [inline backend/src/test/java/de/neuefische/capstone/backend/controller/UserControllerTest.java:34]: Remove unneeded MockBean

**Review by MJaschewski — CHANGES_REQUESTED** [review]
Remove some unused conent from branch

File: backend/src/test/java/de/neuefische/capstone/backend/controller/UserControllerTest.java (hunk 1 above)
Comment (chris-diesner) [reply backend/src/test/java/de/neuefische/capstone/backend/controller/UserControllerTest.java:34]: ...but in case of the webenvironment u were right :)

**Review by MJaschewski — APPROVED** [review]
Looks great!

**Pull request merged**